neo/Ledger/MemoryPool.cs

```diff
@@ -4,162 +4,378 @@
 using System.Collections.Concurrent;
 using System.Collections.Generic;
 using System.Linq;
-using System.Numerics;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using Neo.Persistence;
 
 namespace Neo.Ledger
 {
     internal class MemoryPool : IReadOnlyCollection<Transaction>
     {
-        private class PoolItem
+        private class PoolItem : IComparable<PoolItem>
         {
             public readonly Transaction Transaction;
             public readonly DateTime Timestamp;
+            public readonly Fixed8 FeePerByte;
 
             public PoolItem(Transaction tx)
             {
                 Transaction = tx;
                 Timestamp = DateTime.UtcNow;
+                FeePerByte = Transaction.FeePerByte;
```

Comment:
I guess this line needs to go out again... do you think it's better to revert all my past commits? Sorry for the trouble man, I was just trying to help hahahah

Reply:
no problem